src/routes/login.ts

```diff
@@ -66,9 +66,8 @@ export const createLoginRoute: RouteCreator =
       .getLoginFlow({ id: flow, cookie: req.header("cookie") })
       .then(({ data: flow }) => {
         // Render the data using a view (e.g. Jade Template):
-
         const initRegistrationQuery = new URLSearchParams({
-          return_to: return_to.toString(),
+          return_to: flow.return_to || return_to.toString(),
```

Comment:
I think that `return_to` (which is coming from the query parameters) should override the flow return to. What do you think? Otherwise people might set `/login?return_to=...` but because a return to is already available from a previous flow, it has no effect.

Reply:
Okay yeah that makes sense. It will be more likely that a different flow requires a different return_to URL.